db/seeding/generator.js

```diff
@@ -0,0 +1,173 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable func-names */
+const fs = require('fs');
+const faker = require('faker');
+
+const attractionsLines = 10000000;
+const imagesLines = 10000000;
+const picturesLines = 30000000;
```

Comment:
I think you only use these variables once, each. Might just be best to set the variable i to the number instead of the variable assigned to the number. Can make fore cleaner code but only 3 lines and probably irrelevant.